src/app/routes/assets/models/algorithms.enum.ts

```diff
@@ -1,3 +1,4 @@
 export enum AlgorithmsEnum {
```

Comment:
I know it was already like this, but would consider renaming this to `KeyTypesEnum` as I think it is more accurate than `AlgorithmsEnum`
```suggestion
export enum KeyTypesEnum {
```